src/vi/truyenqq/src/eu/kanade/tachiyomi/extension/vi/truyenqq/TruyenQQ.kt

```diff
@@ -101,7 +99,7 @@ class TruyenQQ : ParsedHttpSource() {
         author = info.select(".org").joinToString { it.text() }
         genre = document.select(".list01 li").joinToString { it.text() }
         description = document.select(".story-detail-info").textWithLinebreaks()
-        thumbnail_url = document.select("img[itemprop=image]").attr("abs:src")
+        thumbnail_url = document.selectFirst("img[itemprop=image]")!!.attr("abs:src")
```

Comment:
`thumbnail_url` is nullable
```suggestion
        thumbnail_url = document.selectFirst("img[itemprop=image]")?.attr("abs:src")
```

Reply:
@dejavui line 54 was good, it was line 102 that needed changing. And it's supposed to be from `!!` to `?`.